src/fetchers/top-languages-fetcher.js

```diff
@@ -33,9 +33,12 @@ const fetcher = (variables, token) => {
   );
 };
 
-async function fetchTopLanguages(username) {
+async function fetchTopLanguages(username, amount=5) {
   if (!username) throw Error("Invalid username");
 
+  amount = parseInt(amount)
+  if (amount > 10 || amount < 1) throw Error("Invalid number of languages")
```

Comment:
Don't need to throw error, just clamp it to the value between 1 & 5

use the clampValue function from the utils.js

```js
const {
  clampValue,
} = require("../src/common/utils");
```